apps/src/templates/certificates/petition/PetitionHelpers.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {isEmail, isInt} from '@cdo/apps/util/formatValidation';
+import FieldGroup from './FieldGroup';
+import {forEach, has} from 'lodash';
+
+export const buildControlledFieldGroup = (
```

Comment:
could this be a react component instead?

Reply:
Brilliant. I made a ControlledFieldGroup component––not sure it's exactly what you're looking for, but it's certainly cleaner than it was. Thoughts about it now?